elements/pfe-progress-indicator/test/pfe-progress-indicator_test.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
+    <script src="/components/@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
+    <script src="/components/web-component-tester/browser.js"></script>
+    <script type="module" src="../pfe-progress-indicator.js"></script>
+  </head>
+  <body>
+
+    <pfe-progress-indicator>
+      This is the element content.
+    </pfe-progress-indicator>
+
+    <script>
+      suite('<pfe-progress-indicator>', () => {
+        test('it should upgrade', () => {
+          assert.instanceOf(document.querySelector('pfe-progress-indicator'), customElements.get("pfe-progress-indicator", 'pfe-progress-indicator should be an instance of pfeProgressIndicator'));
+        });
+
+        // you need to write more tests!!!
```

Comment:
@Djfaucette Just wanted to flag this file as needing more tests.  Not sure if this branch is WIP or ready for review though.